js/web/lib/wasm/session-handler.ts

```diff
@@ -113,7 +113,7 @@ export class OnnxruntimeWebAssemblySessionHandler implements SessionHandler {
   loadModel(model: Uint8Array): void {
     const wasm = getInstance();
     if (!ortInit) {
-      wasm._OrtInit();
+      wasm._OrtInit(env.wasm.numThreads!, 2 /* LoggingLevel::Warning */);
```

Comment:
Add a logging level into Flags?

Reply:
will be separated PR together with SessionOptions/RunOptions support